tests/unit/components/sl-date-range-picker-test.js

```diff
@@ -0,0 +1,352 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-date-range-picker', 'Unit | Component | sl date range picker', {
+    needs: [
+        'component:sl-date-picker'
+    ],
+    unit: true
+});
+
+test( 'Default classNames are present', function( assert ) {
+    this.subject();
+
+    assert.ok(
+        this.$().hasClass( 'sl-date-range-picker' ),
+        'Default rendered component has class "sl-date-range-picker"'
+    );
+});
+
+test( 'Change focus to end date input upon start date change', function( assert ) {
+    assert.expect( 1 );
+
+    this.subject();
+
+    this.$( '.sl-daterange-end-date input' ).on( 'focus', () => {
+        assert.ok( true, 'End date input receives focus upon start date change');
+    });
+
+    Ember.run( () => {
+        this.$( '.sl-daterange-start-date input' ).trigger( 'change' );
+    });
+});
+
+test( 'Earliest end date is the based on min date and start date', function( assert ) {
+    let component = this.subject();
+
+    assert.strictEqual(
+        component.get( 'earliestEndDate' ),
+        null
+    );
+
+    component.set( 'minDate', '01/01/2001' );
+
+    assert.equal(
+        component.get( 'earliestEndDate' ),
+        '01/01/2001'
+    );
+
+    component.set( 'startDateValue', '01/01/2015' );

```

Comment:
RESOLVED (since is working and will be replaced in 1.13 work going to ignore this recommendation)